### PR TITLE
Remove redundant log message

### PR DESCRIPTION
**Pull request by deldesir**

### CODE REVIEW

File: cps/tasks/download.py [hunk 1]
```diff
@@ -30,7 +30,6 @@ def __init__(self, task_message, media_url, original_url, current_user_name, she
     def run(self, worker_thread):
         """Run the download task"""
         self.worker_thread = worker_thread
-        log.info("Starting download task for URL: %s", self.media_url)
```
Comment (holta) [inline cps/tasks/download.py:33]: > log.info("Starting download task for URL: %s", self.media_url)

Is this line _always_ useless?

(Say why if so?!)

Comment (deldesir) [reply cps/tasks/download.py:33]: The task is added with a similar message.

Comment (holta) [reply cps/tasks/download.py:33]: Clarify which code line# (line number) adds the similar message please?

Comment (deldesir) [reply cps/tasks/download.py:33]: https://github.com/iiab/calibre-web/blob/0264fe555da911030adb31342fa55872895fc5c7/cps/tasks/download.py#L159C10-L159C53

File: cps/tasks/download.py [hunk 2]
```diff
@@ -157,7 +156,7 @@ def name(self):
         return N_("Download")
 
     def __str__(self):
-        return f"Download task for {self.media_url_link}"
+        return f"Download task for {self.media_url}"
```
Comment (deldesir) [inline cps/tasks/download.py:159]: https://github.com/iiab/calibre-web/blob/0264fe555da911030adb31342fa55872895fc5c7/cps/tasks/download.py#L159C10-L159C53
```suggestion
        return f"Download task for {self.media_url}"
```

File: cps/tasks/download.py (hunk 1 above)
Comment (deldesir) [reply cps/tasks/download.py:33]: https://github.com/iiab/calibre-web/blob/e56727ccb456deb19d1ae7a2499130a7c425e51f/cps/services/worker.py#L91

**Pull request merged**